qt/scientific_interfaces/Muon/MuonAnalysis.cpp

```diff
@@ -361,16 +361,20 @@ void MuonAnalysis::setChosenGroupAndPeriods(const QString &wsName) {
   const auto wsParams =
       MuonAnalysisHelper::parseWorkspaceName(wsName.toStdString());
 
-  const QString &groupToSet = QString::fromStdString(wsParams.itemName);
   const QString &periodToSet = QString::fromStdString(wsParams.periods);
-  const auto &groups = m_dataSelector->getChosenGroups();
   const auto &periods = m_dataSelector->getPeriodSelections();
-  if (!groups.contains(groupToSet)) {
-    m_uiForm.fitBrowser->setChosenGroup(groupToSet);
-  }
+
   if (!periodToSet.isEmpty() && !periods.contains(periodToSet)) {
     m_uiForm.fitBrowser->setChosenPeriods(periodToSet);
   }
+  return; /*
+  const QString &groupToSet = QString::fromStdString(wsParams.itemName);
```

Comment:
Is this code commented out for a reason?

There is no note to explain this, in which case it should either be uncommented or deleted.

It will only cause confusion in the codebase laster.

Reply:
I will delete this. I thought it was for periods and might be needed later.